src-tauri/src/domain/impls.rs

```diff
@@ -63,19 +63,10 @@ impl Document {
 
     pub fn harvard_reference(&self) -> String {
         match &self {
-            &Document::BR187(_) => " (BR 187, 2nd edition, External fire spread: building separation and boundary distances".to_string(),
-            &Document::PD7974(part) => match part {
-                Some(part) => format!("PD7974, {}", part.friendly_reference()),
-                None => "PD7974".to_string(),
-            },
-            &Document::SFPEHandbook(c) => match c {
-                Some(c) => format!("SFPE Handbook, {}", c.friendly_reference()),
-                None => "SFPE Handbook".to_string(),
-            },
-            &Document::IntroductionToFireDynamics(c) => match c {
-                Some(c) => format!("Introduction to Fire Dynamics, {}", c.friendly_reference()),
-                None => "Introduction to Fire Dynamics".to_string(),
-            },
+            &Document::BR187(_) => "Chitty,R., 2014. External fire spread: Building Separation and Boundary Distances (BR 187). 2nd edition".to_string(),
+            &Document::PD7974(_) => "BSI, 2019. PD 7974-1: Application of Fire Safety Engineering Principles to the Design of Buildings – Initiation and Development of Fire within the Enclosure of Origin (Sub-system 1). BSI".to_string(),
```

Comment:
This only works for part 1. The reference enums are built in a way that they contain inner details (the part). So I would do this like this:

```suggestion
            &Document::PD7974(part) => format!("BSI, 2019. PD 7974-{}: Application of Fire Safety Engineering Principles to the Design of Buildings – {}. BSI", part.number, part.title),
```

then add the functions name() and title() to the inner parts of PD7974.

Reply:
Where do I define name and title? Would that be line 101?
@JamieMaclean

Reply:
No. Line 34 no?